pkg/sql/pgwire/testdata/pgtest/connection_params

```diff
@@ -0,0 +1,55 @@
+# Change the application name.
+# NOTE: the order is different than postgres, because we execute the statement
```

Comment:
Are you sure this is the reason? The commit message is more unsure. (I'm also unsure, I just don't want to have an incorrect statement here.)